tests/virt_autotest/libvirt_extend_storage_lvm.pm

```diff
@@ -132,6 +132,8 @@ sub create_volume_group {
     # Create new disk partition for LVM volumes
     record_info "Create new disk partition for LVM volumes";
     assert_script_run 'echo -e "g\nn\n\n\n+20G\nt\n8e\np\nw" | fdisk ' . $lvm_disk_name;
+    # Enable NVME Hard Drive Support
+    $lvm_disk_name = ($lvm_disk_name =~ "nvme") ? "${lvm_disk_name}p" : $lvm_disk_name;
```

Comment:
Does this apply to all nvme cases ?

Reply:
As the same, yes, this changed would be applied to our all testing NVME Hard Drive. But, Still need more VR to validate.